src/lib/device/physical_dev.cpp

```diff
@@ -220,8 +220,10 @@ void PhysicalDev::submit_batch() { m_drive_iface->submit_batch(); }
 
 //////////////////////////// Chunk Creation/Load related methods /////////////////////////////////////////
 void PhysicalDev::format_chunks() {
-    m_chunk_info_slots = std::make_unique< sisl::Bitset >(hs_super_blk::chunk_info_bitmap_size(m_dev_info));
+    m_chunk_info_slots = std::make_unique< sisl::Bitset >(std::max(1u, hs_super_blk::max_chunks_in_pdev(m_dev_info)));
```

Comment:
I didn't understand the comment, why would I need to alignment_size here

Reply:
if you specified alignment when initializing  the bitset ,  the serialize() is free, otherwise it is a copy.
c.f https://github.com/eBay/sisl/blob/03282536e309252d85319bebf4b227be0be6f9fe/include/sisl/fds/bitset.hpp#L647

Reply:
now , when we  call `max_chunks_in_pdev`, we calculate the max_chunks_num using the following code.
```
(dinfo.dev_size - 1) / min_chunk_size + 1;
```

however , we have superblk and vdevinfo in the starting part of a pdev. so should we take into account this when calculating?

Reply:
this is jut the max_chunk we can have (#bits in the bitmap). Space is another limitation , which checks when creating each chunk